Attendance/Attendance/Course.m

```diff
@@ -22,8 +22,10 @@ - (instancetype)init
 -(NSString *)roleCall{
     
     // TODO: Complete this method
-    NSString *toReturn = @"";
-    
+    NSMutableString *toReturn = [[NSMutableString alloc] init];
+    for (Student *student in _students) {
+        [toReturn appendFormat:@"%@",student];
```

Comment:
Placing a new line between each student would improve the output: `[toReturn appendFormat:@"%@\n",student];`